dictdiffer/__init__.py

```diff
@@ -65,6 +66,7 @@ def diff(first, second, node=None, ignore=None, path_limit=None, expand=False):
     :param path_limit: List of path limit tuples or dictdiffer.utils.Pathlimit
                        object to limit the diff recursion depth
     :param expand: Expands the patches
+    :param tolerance: threshold to consider 2 values identical
```

Comment:
The doc and changelog are updated.
What do you mean by 'add mention to a commit message' ?